app/views/pages/domains.html.erb

```diff
@@ -55,7 +55,7 @@
     <p class="top_toc"><%=link_to('Back to Top', anchor: '#toc-top') %></p>
 
     <h3 class="about_h3" id="5toc">The domain derivative download</h3>
-    <p class="about_p">By clicking on the "Domains" button, users are provided with a text file that will be named something like 7485-fullurls.txt. You can open this file up in your text editor of choice. By default on MacOS, it will open in TextEdit and look like this:</p>
+    <p class="about_p">By clicking on the "Domains" button, users are provided with a text file that will be named something like 7485-fullurls.csv. You can open this file up in your text editor of choice. By default on MacOS, it will open in TextEdit and look like this:</p>
```

Comment:
`text file` -> `comma-separated value file` 
`it will open in TextEdit and look like this` -> `it will open in a spreadsheet program (Excel or Numbers) and look like this`